bentoml/utils/flask_ngrok.py

```diff
@@ -0,0 +1,94 @@
+import atexit
```

Comment:
Any reason not to just use the flask-ngrok package directly?

Reply:
flask-ngrok package in pypi is not updated and not maintained. I tried to use flask-ngrok package directly, but latest code is not available in pypi

Reply:
got you, that make sense